Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture_test.py

```diff
@@ -190,3 +240,50 @@ def __next__(self):
             raise StopIteration()
         self.has_returned_dict_reader = True
         return self.dict_reader
+
+
+@pytest.mark.parametrize('indicator_type, risk_rules, service, expected_err_msg',
+                         [('url', 'dhsAis', 'fusion',
+                           "You entered a risk rule but the 'connectApi' service is not chosen."),
+                          ('url', 'dhsAi', 'connectApi', "The given risk rule: dhsAi does not exist"),
+                          ('url', 'dhsAis,phishinUrl', 'connectApi', "The given risk rule: phishinUrl does not exist")
+                          ])
+def test_risk_rule_validations(mocker, indicator_type, risk_rules, service, expected_err_msg):
+    """
+    Given:
+     - Recorded Future Feed client initialized with URL indicator type and:
+      1. 'fusion' service, and a valid risk rule.
+      2. 'connectApi' service, and an invalid risk rule.
+      3. 'connectApi' service, and a comma separated list of a valid risk rule (the first) and an invalid risk rule
+        (the second).
+
+     - Mock response of the 'FeedRecordedFuture.return_error function', and the
+      'FeedRecordedFuture.Client.get_risk_rules' command.
+
+    When:
+     - Running the 'Client.run_parameters_validations'.
+
+    Then:
+     - Verify the right error message appears for each case:
+     1. Error message for setting 'connectApi' service.
+     2. Error message for invalid risk rule.
+     3. Error message for invalid risk rule on the second risk rule in the risk rules list.
+    """
+    mocker.patch('FeedRecordedFuture.return_error', side_effect=mock_return_error)
+    from FeedRecordedFuture import Client
+    client = Client(indicator_type=indicator_type, api_token='123', risk_rule=risk_rules, services=service)
+
+    mocker.patch('FeedRecordedFuture.Client.get_risk_rules',
+                 return_value={'data': {'results': [{'name': 'dhsAis'}, {'name': 'phishingUrl'}]}})
+    with pytest.raises(Exception) as e:
```

Comment:
i think you can catch `SystemExit` and avoid mocking `return_error`

Reply:
changed that.